linkerd/app/outbound/src/http/logical/policy/route.rs

```diff
@@ -22,26 +22,35 @@ pub(crate) struct Matched<M, P> {
 }
 
 #[derive(Clone, Debug, PartialEq, Eq, Hash)]
-pub(crate) struct Route<T, F> {
+pub(crate) struct Route<T, F, E> {
     pub(super) parent: T,
     pub(super) addr: Addr,
     pub(super) meta: Arc<policy::Meta>,
     pub(super) filters: Arc<[F]>,
     pub(super) distribution: BackendDistribution<T, F>,
+    pub(super) failure_policy: E,
```

Comment:
based on the naming --- is this just a response classifier, or is it intended to also eventually represent actions taken on failures (e.g. updating a failure counter, etc)?